cloudnet-api/cloudnet-api-bridge/src/main/java/eu/cloudnetservice/cloudnet/v2/bridge/CloudServer.java

```diff
@@ -106,24 +106,22 @@ public static CloudServer getInstance() {
 
     public void updateDisable() {
         List<String> list = new CopyOnWriteArrayList<>();
```

Comment:
Can this CopyOnWriteArray be replaced by an ArrayList ?

Reply:
I left the CopyOnWriteArrayList Objects as they are, bc I think they were being implemented for the specific use, to iterate over Thread safe. Two options:
```suggestion
        List<String> list = Bukkit.getOnlinePlayers().stream().map(HumanEntity::getName).collect(Collectors.toCollection(
            CopyOnWriteArrayList::new));
```

```suggestion
        List<String> list = Bukkit.getOnlinePlayers().stream().map(HumanEntity::getName).collect(Collectors.toList());
```

Reply:
Well the CopyOnWriteArrayList is slower and not as fast as a normal ArrayList and does not necessarily contribute to thread security especially in this context